### PR TITLE
Product details: Styling and sizes comparison

### LINKED ISSUE #101: Product details
Section displayed in horizontal accordion (the section being the fragment) with multiple variations.

Variation 1: 1 product image (right) and text on the left
Screenshot 2023-06-23 at 11 10 29 AM

Variation 2: Two product images with text left and right
Screenshot 2023-06-23 at 11 11 02 AM

Variation 3: Multiple product images with single-line text on top
Screenshot 2023-06-23 at 11 11 36 AM

Variation 4: 2 product images with stats
Screenshot 2023-06-23 at 11 10 43 AM

Note how all have the same bottom background, and have some transparency to see the video behind hit.

Original Page: https://sgcreative.southernglazers.com/bloominbrands/p/7

### PR DESCRIPTION
**Pull request by badvision**
## Issue

Fix #101 

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/drafts/brobert/p7
- After: https://product-details--sgws-creative--hlxsites.hlx.page/drafts/brobert/p7

## 📝 Description:

This PR adds a lot of additional styling support for vertical accordion/tabs as well as the table background style.  The comparison/size variant is also implemented.

### CODE REVIEW

File: blocks/comparison/comparison.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/* select comparison-wrapper that has a child having .wood-table class */
+.vs-active.wood-table:after {
+    content: " ";
+    display: block;
+    width: 100%;
+    margin-top: calc(-9vw);
+    height: 10vw;
+    background: url(https://media-s3-us-east-1.ceros.com/southern-glazers/images/2023/04/18/1ec7669e44ab282115f474839b13461c/wood-table-top-on-light-blur-background-empty-brown-wood-table-image.jpg?imageOpt=1&fit=bounds&width=2600&crop=4000,539,x0,y1764);
```
Comment (tmathern) [inline blocks/comparison/comparison.css:8]: We'll need to be clear that that "bar top table" won't be editable. (It's a design element).

File: blocks/comparison/comparison.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    background-size: cover;
+    z-index: index 10;
+}
+
+.comparison-wrapper {
+    flex-grow: 1;
+    max-height: calc(39.5vw);
+    z-index:20;
```
Comment (tmathern) [inline blocks/comparison/comparison.css:16]: Why 20? (And not 10, for instance?)

File: blocks/comparison/comparison.css (hunk 1 above)
Comment (badvision) [reply blocks/comparison/comparison.css:8]: Agreed.  It should be noted as such in the authoring guide.

File: blocks/comparison/comparison.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
+/* select comparison-wrapper that has a child having .wood-table class */
+.vs-active.wood-table::after {
+    content: " ";
+    display: block;
+    width: 100%;
+    margin-top: calc(-9vw);
```
Comment (tmathern) [inline blocks/comparison/comparison.css:6]: Do we really need `calc` here?

File: blocks/comparison/comparison.css [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    height: 10vw;
+    background: url("/styles/wood-table-top.avif");
```
Comment (tmathern) [inline blocks/comparison/comparison.css:8]: I would prefer a webp version, but... :) (I know the originals are all avif).

File: blocks/comparison/comparison.css [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    align-items: flex-start;
+}
+
+.comparison.size.block > div > div:nth-child(2n) {
+    margin-top: 10px;
+}
+
+@media screen and (min-width: 1540px) {
+    .comparison.size.block {
+        padding-top:8vw;
```
Comment (tmathern) [inline blocks/comparison/comparison.css:41]:
```suggestion
        padding-top: 8vw;
```

File: blocks/vertical-selector/vertical-selector.js [hunk 6]
```diff
@@ -61,7 +61,7 @@ function handleSelectorClick(scope, index) {
   const displayedViewer = viewer.querySelector('div.vs-active');
 
   // Ignore the click if already selected.
-  if (displayedViewer.classList.contains(`fragment-viewer-${index}`)) {
+  if (displayedViewer && displayedViewer.classList.contains(`fragment-viewer-${index}`)) {
```
[the remark this answers is not in the document]
Comment (badvision) [reply blocks/vertical-selector/vertical-selector.js:64]: Old habits die hard.  Please remember I started JS before ?. operator was a thing (specifically back in '97)

**Pull request merged**